test/Driver/options-interpreter.swift

```diff
@@ -27,9 +27,8 @@
 // RUN: %swift_driver -### -target x86_64-apple-macosx10.9 -L/foo/ -L/bar/ %s | %FileCheck -check-prefix=CHECK-L2 %s
 // CHECK-L2: # DYLD_LIBRARY_PATH={{/foo/:/bar/:[^:]+/lib/swift/macosx$}}
 
-// RUN: env DYLD_LIBRARY_PATH=/abc/ SDKROOT=/sdkroot %swift_driver_plain -### -target x86_64-apple-macosx10.9 -L/foo/ -L/bar/ %s | %FileCheck -check-prefix=CHECK-L2-ENV %s
-// CHECK-L2-ENV: # DYLD_LIBRARY_PATH={{/foo/:/bar/:[^:]+/lib/swift/macosx:/sdkroot/usr/lib/swift:/abc/$}}
-
+// RUN: not env DYLD_LIBRARY_PATH=/abc/ SDKROOT=/sdkroot %swift_driver_plain -### -target x86_64-apple-macosx10.9 -L/foo/ -L/bar/ %s 2>&1 | %FileCheck -check-prefix=CHECK-L2-ENV %s
+// CHECK-L2-ENV: error: unable to find the 'arclite' library
```

Comment:
These tests are checking for other behavior that we want to preserve, aren't they? I think it would be better to put a dummy arclite library in the SDK during this test and create a new test file for this arclite error. (Alternatively, if we switch to a warning instead of an error, you might be able to simultaneously test the new warning and the existing behavior.)

Reply:
Just changed to warning, so test simultaneously :))